README.md

```diff
@@ -415,6 +415,8 @@ At some point, you will likely want to pay yourself a salary. The outline of the
 
 5. At the beginning of the following month, send in the employer declaration to Skatteverket, declaring how much salary you've paid out.
 
+6. The month after that you need to pay Skatteverket in the income and employer taxes that you declared.
```

Comment:
Can this be rephrased? Do you mean that the income taxes can be payed in the next month?

Reply:
Easier with an example?

You work in January and pay salary on the 25th like most companies. As an employer you have to report salaries by mid February to Skatteverket. Then you have to pay the employer contributions and income tax that correspond to that report by mid March

Reply:
> Easier with an example?
> 
> You work in January and pay salary on the 25th like most companies. As an employer you have to report salaries by mid February to Skatteverket. Then you have to pay the employer contributions and income tax that correspond to that report by mid March

Got it! What about:

"You only have to pay the taxes to Skatteverket the month after you paid the salary. Example:
You work in January and pay salary on the 25th like most companies. As an employer you have to report salaries by mid February to Skatteverket. Then you have to pay the employer contributions and income tax that correspond to that report by mid March."

Reply:
But you pay the taxes 2 months after you pay the salaries. But you report them 1 month after.

Reply:
Actually, i think i am wrong. I think you both have to report and pay the taxes 2 months (actually about 1.5 months) after you pay them. I get confused by this delay so i always report them on the straight after the end of the month to avoid confusion